### ISSUE TITLE
Number of epochs for benchmark dataset

### ISSUE THREAD

**Issue by Chen-Cai-OSU** [Accenture/AmpliGraph#98]
Hello,

First, thanks a lot for such a nice library!

I have a question on the performance reported [here](https://docs.ampligraph.org/en/latest/experiments.html#runtime-performance). I was wondering if 4000 epoch really necessary to achieve the reported result? My machine is not running as fast as yours, and It would be nice to know I can still replicate your result using fewer epochs.

Also, are you able to replicate the results for most of the method you implemented? I just checked fb-237, it seems that for some old method, your result is better than that is reported in the paper.

**Comment by sumitpai** [comment]
Most experiments had early stopping (on validation sets) and it didn't run up to 4000 epochs. We will update the documentation to reflect this. You should get good results on most of the reported datasets in less than 300 epochs. You could try running the some experiments for 4000 epochs with early stopping on validation set. It will stop much before 1000 epochs.

**Comment by sumitpai** [comment]
We get better results for most models with the adversarial loss as described in the [RotatE](https://openreview.net/pdf?id=HkgEQnRqYQ) paper

**Comment by Chen-Cai-OSU** [comment]
Cool! Thanks a lot!

**Comment by sumitpai** [comment]
Need to update docs with early stopping parameters

**Comment by chanlevan** [comment]
Early stopping information have been added to all experiments by appending to the hyperparameters columns.
